Frontend/ui-customer/src/Flow.purs

```diff
@@ -3105,18 +3106,19 @@ metroTicketBookingFlow = do
       if state.props.currentStage == MetroTicketSelection then do
         (SearchMetroResp searchMetroResp) <- Remote.searchMetroBT (Remote.makeSearchMetroReq state.data.srcCode state.data.destCode state.data.ticketCount)
         modifyScreenState $ MetroTicketBookingScreenStateType (\state -> state{data{ searchId = searchMetroResp.searchId}, props { currentStage = GetMetroQuote }})
-        metroTicketBookingFlow
-        else if state.props.currentStage == ConfirmMetroQuote then do
-          (confirmMetroQuoteResp) <- Remote.confirmMetroQuoteBT state.data.quoteId
-          let (MetroTicketBookingStatus metroBookingStatus) = confirmMetroQuoteResp
-          modifyScreenState $ MetroTicketBookingScreenStateType (\state -> state{data{ bookingId = ( (metroBookingStatus.bookingId))}})
-          void $ lift $ lift $ delay $ Milliseconds 3000.0
-          metroTicketPaymentFlow metroBookingStatus.bookingId
-          else do
-            metroTicketBookingFlow  
+      else if state.props.currentStage == ConfirmMetroQuote then do
+        (confirmMetroQuoteResp) <- Remote.confirmMetroQuoteBT state.data.quoteId
+        let (MetroTicketBookingStatus metroBookingStatus) = confirmMetroQuoteResp
```

Comment:
update this to `(MetroTicketBookingStatus confirmMetroQuoteResp) <- Remote.confirmMetroQuoteBT state.data.quoteId`

Then `Line 3111` is not required